Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-alpine
```

Comment:
I think python 3:6 are used in other images due to debian LS is there - but not sure.

Reply:
as the base here is alpine, any debian related things do not really matter... or did i miss the point?!

Reply:
yeah, I really don't have a good answer :) lets just keep it at this version for now and if someone remembers why homeassistant it self is at 3.6 then we can easily revert it.